src/tools/Reporting/Reporting/Reporter.cs

```diff
@@ -89,6 +89,12 @@ private void Init()
                 TimeStamp = DateTime.Parse(environment.GetEnvironmentVariable("PERFLAB_BUILDTIMESTAMP")),
             };
             build.AdditionalData["productVersion"] = environment.GetEnvironmentVariable("DOTNET_VERSION");
+
+            // Additional Data we only want populated if it is available
+            if (environment.GetEnvironmentVariable("MAUI_VERSION") != null)
```

Comment:
nit: would be cleaner using a pattern match rather than a null check I think
```csharp
if (environment.GetEnvironmentVariable("MAUI_VERSION") is string mauiVersion)
{
    build.AdditionalData["mauiVersion"] = mauiVersion;
}
```